test/test_torch.py

```diff
@@ -315,6 +315,15 @@ def test_max(self):
     def test_min(self):
         self._testSelection(torch.min, min)
 
+    # tests whether min/max/median propagate to a single value
+    def test_select_first_equal_backward(self):
```

Comment:
I think test_autograd.py would be better as you test autograd-related stuff.
Also if you add it [here](https://github.com/pytorch/pytorch/blob/f813e7184e6098de7557a7873442428182f29ad4/test/test_autograd.py#L5220) in the generic tests, you will get the cpu/cuda tests for free by just using the device that is given.

Reply:
Thanks! Done